src/Api/Dto/CartItemInput.php

```diff
@@ -14,7 +14,7 @@ final class CartItemInput
     /**
      * @Groups({"cart"})
      */
-    public $quantity = 1;
+    public $quantity;
```

Comment:
I'm afraid this can break current versions of the app that are not sending the quantity yet.

Reply:
We should add a functional test for this. To check if it works **both** when sending the quantity, or not (legacy).

Reply:
You're right, I'll check how to make it work for both.

Reply:
I think you can just let it like this, no? With the default value I mean.

Reply:
Yes thanks, if I let it with the default value it works. I thought that it will always put 1 as quantity but not, it's just a default value.
Reverted :)